lab03-kaylee/fs-textData.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
This module looks good. The lab didn't require a specific directory structure, so no points lost for this, but in the future, any modules that define your functions like this one should be put in a ``lib`` folder.